WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -159,6 +156,20 @@ private extension ItemListView {
             })
         }
     }
+
+    @ViewBuilder
+    func listRow(item: any POSDisplayableItem) -> some View {
+        switch item {
+        case is any POSOrderableItem:
+            Button(action: {
+                posModel.addToCart(item as! (any POSOrderableItem))
```

Comment:
I wish it was possible to unwrap this a bit better, but it can't actually crash.

Reply:
Agree. There's little value from `switch` in this case, we are not switching through enum cases and just checking a type.

Reply:
Thanks 😊 

I'd tried to do this and got the error `Pattern variable binding cannot appear in an expression`, which I thought meant I couldn't do this type of unwrapping in a viewbuilder, but actually just meant I couldn't use the same syntactical sugar for a typecasting unwrap as we can for a normal unwrap. I.e., this isn't valid:

```
if let item as? any POSOrderableItem { ... }
```

But this is:

```
if let item = item as? any POSOrderableItem { ... }
```